### DISCUSSION TITLE
how to include results from a relation table in a findUnique() query from another table

### DISCUSSION THREAD

**Discussion by vladandrei0** [prisma/prisma#4500] [Q&A] [answered]
Hello, 
I have this model

```prisma
model User {
   id    Int     @id @default(autoincrement())
   fbuid String? @unique
   email String  @unique
}

model Sheet {
   id      Int    @id @default(autoincrement())
   name    String
   theme   Theme  @relation(fields: [themeId], references: [id])
   themeId Int
}

model User_Sheet {
   userId Int
   sheetId Int
   liked  Boolean @default(false)

   @@id([userId, fisaId])
}

model Theme {
   id    Int     @id @default(autoincrement())
   name  String
   Sheet Sheet[]
}
```

and this query:

```ts
const sheet = await prisma.sheet.findUnique({
            where: { id: Number(id) },
            include: {       
                tema: true
            }
        })

const user_sheet= await prisma.user_sheet.findUnique({
            where: {
                userId_sheetId: {
                    userId: Number(fbid),
                    sheetId: Number(id)
                }
            }
        })
```

how can I include results from User_Sheet? I have both the User id and Sheet id present at the time of the query.
I use Next js, and I am inside a getServerSideProps function.
Can I do this with one query and include somehow the relation table or do I need to keep both?
Many thanks

**Comment by ryands17** [comment, marked as answer]
Hey @vladandrei0 👋 

Your `User_Sheet` model doesn't include relations to the `User` and `Sheet` tables. So in this case you would need to change your model to the following:

```prisma
model User {
  id     Int          @id @default(autoincrement())
  fbuid  String?      @unique
  email  String       @unique
  sheets User_Sheet[]
}

model Sheet {
  id      Int          @id @default(autoincrement())
  name    String
  theme   Theme        @relation(fields: [themeId], references: [id])
  themeId Int
  users   User_Sheet[]
}

model User_Sheet {
  userId  Int
  sheetId Int
  liked   Boolean @default(false)

  User  User  @relation(fields: [userId], references: [id])
  Sheet Sheet @relation(fields: [sheetId], references: [id])
  @@id([userId, sheetId])
}

model Theme {
  id    Int     @id @default(autoincrement())
  name  String
  Sheet Sheet[]
}
```

Now you can directly query with `include` for all the Users and Sheets as follows:

```ts
  const user_sheet = await prisma.user_Sheet.findUnique({
    where: {
      userId_sheetId: {
        userId: Number('13123132'),
        sheetId: Number('122132'),
      },
    },
    include: { User: true, Sheet: true },
  })
```

You can read more about relations here: https://www.prisma.io/docs/concepts/components/prisma-schema/relations

**Reply by vladandrei0** [reply]
I see what u did there!!  works like a charm 😁 Thanks a lot!!!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
